<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_superior">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>781</width>
         <height>48</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_minimizar">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>minimizar.png</normaloff>minimizar.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cerrar">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>cerrar.png</normaloff>cerrar.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_contenido">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(28, 113, 216);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>10</y>
         <width>381</width>
         <height>471</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QListWidget" name="list_musica">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>331</width>
          <height>261</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(26, 95, 180);</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>330</y>
          <width>321</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="lbl_titulo">
           <property name="styleSheet">
            <string notr="true">color: rgb(26, 95, 180);</string>
           </property>
           <property name="text">
            <string>TITULO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_artista">
           <property name="styleSheet">
            <string notr="true">color: rgb(26, 95, 180);</string>
           </property>
           <property name="text">
            <string>ARTISTA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_genero">
           <property name="styleSheet">
            <string notr="true">color: rgb(26, 95, 180);</string>
           </property>
           <property name="text">
            <string>GENERO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_anio">
           <property name="styleSheet">
            <string notr="true">color: rgb(26, 95, 180);</string>
           </property>
           <property name="text">
            <string>AÃ‘O</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>4</y>
         <width>381</width>
         <height>481</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>110</y>
             <width>311</width>
             <height>111</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="grafica_layout"/>
          </widget>
          <widget class="QPushButton" name="btn_buscar">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>181</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(26, 95, 180);
font: 75 italic 11pt &quot;Serif&quot;;
border: 2px solid rgb(26, 95, 180);</string>
           </property>
           <property name="text">
            <string> Buscar Musica</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>musica.png</normaloff>musica.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QSlider" name="slider_volumen">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>150</y>
             <width>131</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(26, 95, 180);</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_volumen">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>180</y>
             <width>131</width>
             <height>25</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(26, 95, 180);</string>
           </property>
           <property name="text">
            <string>50</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QSlider" name="slider_tiempo">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>10</y>
             <width>241</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(26, 95, 180);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>150</y>
             <width>221</width>
             <height>48</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btn_sil">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>silencio.png</normaloff>silencio.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_bajvol">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>bajar-volumen.png</normaloff>bajar-volumen.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_subvol">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>sube-el-volumen.png</normaloff>sube-el-volumen.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>361</width>
             <height>48</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btn_ant">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>back.png</normaloff>back.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_pausar">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>pause.png</normaloff>pause.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_reproducir">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>play.png</normaloff>play.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_sig">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>forward.png</normaloff>forward.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>361</width>
             <height>22</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="indicador_posicion">
              <property name="styleSheet">
               <string notr="true">color: rgb(26, 95, 180);</string>
              </property>
              <property name="text">
               <string>00:00</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="indicador_tiempo">
              <property name="styleSheet">
               <string notr="true">color: rgb(26, 95, 180);</string>
              </property>
              <property name="text">
               <string>00:00</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
